test.py

```diff
@@ -36,6 +36,7 @@
 from test.pylib.artifact_registry import ArtifactRegistry
 from test.pylib.host_registry import HostRegistry
```

Comment:
> [test.py] Add resource consumption metrics

Please enrich the patch changelog with details.

Reply:
By changelog, you mean to add information to the PR description? Because I was unable to find the changelog file dedicated to test.py.